packages/admin-portal/src/components/iaas/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { Intent } from '@reapit/elements'
+import { PipelineRunnerType } from '@reapit/foundations-ts-definitions'
+
+export const buildStatusToIntent = (status: string): Intent => {
```

Comment:
can we SRP these please? Looks like they've been duplicated? Best to keep them in one place so changes will be reflected everywhere

Reply:
Yep, agreed, was just being lazy